src/NzbDrone.Api/Calendar/CalendarFeedModule.cs

```diff
@@ -92,6 +93,10 @@ private Response GetCalendarFeed()
                 ProductId = "-//sonarr.tv//Sonarr//EN"
             };
 
+            var calendarName = "Sonarr TV Schedule Calendar";
```

Comment:
Can you update this to `Sonarr TV Schedule`? Calendar is implied and best left off I think.

Reply:
No problem. Will update and ping when I've updated the PR.

Reply:
@markus101 Updated the default name. Should be good to merge.